solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,668 @@
+##############################################################################
+# Common
+##############################################################################
+
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API key used to provision resources."
+  sensitive   = true
+}
+
+variable "existing_secrets_manager_crn" {
+  type        = string
+  description = "The CRN of secrets manager instance to create the secret engine in."
+  nullable    = false
+}
+
+variable "prefix" {
```

Comment:
this needs to be updated to comply with our latest prefix structure. Please check with @maheshwarishikha as its documented in a git issue I believe